clean.py

```diff
@@ -4,14 +4,15 @@
 import os
 
 directory = 'figures'
-logfile = 'rus.log'
+logfile = 'test.log'
+enc = 'Latin-1'
```

Comment:
The encoding option is a sensible improvement, however I would prefer to let the default be UTF8 since I think it's still more common.